enterprise/server/scheduling/task_router/task_router.go

```diff
@@ -76,20 +84,23 @@ func (tr *taskRouter) RankNodes(ctx context.Context, cmd *repb.Command, remoteIn
 		nodes[i], nodes[j] = nodes[j], nodes[i]
 	})
 
-	if cmd == nil {
-		return nodes
-	}
-	preferredNodeLimit := getPreferredNodeLimit(cmd)
-	if preferredNodeLimit == 0 {
+	params := getRoutingParams(ctx, tr.env, cmd, remoteInstanceName)
+	strategy := tr.selectRouter(params)
+	if strategy == nil {
+		log.Info("no routing strategy applied to the routing params")
```

Comment:
in prod we run with

```  log_level: "info"```

so this might log quite often (for all actions which neither are workflows or have your new prop set) -- is that right and what you want?

Reply:
Oh yeah, I had this here from an earlier iteration where there was a fallback, but it doesn't make sense now so I removed it.